torch_xla/experimental/xla_sharded_tensor.py

```diff
@@ -97,7 +94,7 @@ def __new__(cls, elem: torch.Tensor, *args, **kwargs):
 
   # Shards on the devices are materialized/available after the lazy
   # execution of the SPMDPartitioned HLO graph. Each XLAShard points
-  # to torch.Tensor (xla::device_data). The shards represent a snapshot
+  # to torch.Tensor (xla::device_data). The shards represent a snapshot on CPU
```

Comment:
nit. Could we update the comment here, "execution of the partitioned HLO graph. Each XLAShard points to torch.Tensor. The shards represent a snapshot on CPU, detached from the global tensor." ?